DOM/to-do-app/JS/script.js

```diff
@@ -24,6 +25,17 @@ document.getElementById("filterByStatus").addEventListener("change", () => {
   showAllTasks(data.tasks);
 });
 
+document.getElementById("searchButton").addEventListener("click", () => {
+  createNewTask();
+  showAllTasks(data.tasks);
```

Comment:
why we need to pass data to function? can't we take `data.tasks` inside `taskHelpers.js` folder?